dataprofiler/tests/profilers/test_text_column_profile.py

```diff
@@ -299,3 +299,48 @@ def test_merge_timing(self):
             expected_times = defaultdict(float, {'vocab': 6.0})
             self.assertDictEqual(expected_times, profiler3.profile['times'])
 
+    def test_profile_merge_with_different_options(self):
+        # Creating first profiler with default options
+        options = TextOptions()
+        options.max.is_enabled = False
+        options.min.is_enabled = False
+
+        df = pd.Series(
+            ["pancake", "banana", "lighthouse", "aa", "b", "4", "3", "2", "dfd", "2"]
+        ).apply(str)
```

Comment:
why do you apply str to a series full of strings?

Reply:
Precedence. but really, I guess no reason at all. Took it out.